frontend/src/components/ShootListRow.vue

```diff
@@ -265,4 +301,30 @@ export default {
   .labels {
     line-height: 10px;
   }
+
+  .position-relative {
+    position: relative;
+  }
+
+  .stale-overlay {
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    position: absolute;
+    pointer-events: none;
+  }
+
+  .theme--light .stale .stale-overlay {
+    background-color: rgba(255,255,255,0.8)
+  }
+  .theme--dark .stale .stale-overlay {
+    background-color: rgba(0,0,0,0.8)
```

Comment:
![Screenshot 2022-12-05 at 13 58 39](https://user-images.githubusercontent.com/5526658/205643237-a5a5b50a-c07b-4f8e-9034-19f9a9d81480.png)
in dark mode it's a bit too dark

Reply:
<img width="491" alt="Screenshot 2022-12-07 at 15 30 36" src="https://user-images.githubusercontent.com/35373687/206205901-733cad47-7f71-4276-96bb-56143afff7f0.png">

better?

Reply:
yes 👍